hpcgap/lib/read.g

```diff
@@ -28,35 +28,42 @@ fi;
 
 #############################################################################
 ##
-#X  Read transitive groups library
+#X  Checking transitive groups library
 ##
+## Assign TransitiveGroupsAvailable to a dummy function to make it
+## callable, even if the library is unavailable.
+if TestPackageAvailability("TransGrp")=fail then
```

Comment:
can we please change this to `"transgrp"`, to match the other places (same for `PrimgGrp` and `SmallGrp` below). Or, if you prefer, change all other places to use `TransGrp`. Just as long as it is consistent.

Reply:
Agree. I think because LoadPackage and friends is case-insensitive, lowercase everywhere for all packages would be consistent (I do not use uppercase when requesting a package, and I think the same applies to most of users). 

The canonical NAME of the package is the other thing. Here authors may combine upper and lower case as they wish. My preferred spelling would be mixed case, like PrimGrp, SmallGrp, TransGrp in this case.

Reply:
If you'd like the packages to use that spelling for their canonical name, then feel free to submit pull requests to change their `PackageInfo.g` files accordingly -- right now, though, transgrp and smallgrp use an all-lowercase spelling, only PrimGrp does not.